src/components/data-management/SamplesTable.jsx

```diff
@@ -245,13 +254,13 @@ const SamplesTable = forwardRef((props, ref) => {
 
       return {
         key: idx,
-        name: samples[sampleUuid].name,
+        name: sampleData.name,
         uuid: sampleUuid,
         barcodes: barcodesData,
         genes: genesData,
         matrix: matrixData,
-        species: samples[sampleUuid].species || DEFAULT_NA,
-        ...samples[sampleUuid].metadata,
+        species: sampleData.species || DEFAULT_NA,
```

Comment:
do we still use this species here?

Reply:
Oh right, we've removed species. I'll remove this.